canonical-websites/www.ubuntu.com
# update /cloud/plans-and-pricing with new pricing includes

## Done

* remove autopilot from contextual footer
* updated with new includes
* deleted several old includes
* made the OpenStack table col-10 from col-12
* fixed some bad CTAs and google events

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [cloud p&p page](http://0.0.0.0:8001/cloud/plans-and-pricing) and [support p&p page](http://0.0.0.0:8001/support/plans-and-pricing)
- compare to the [cloud p&p copy doc](https://docs.google.com/document/d/1yZ7wClcmxGCqMyyyBJLO66Z3YlpT-4soWLFmJ6uXeMM/edit#heading=h.tp2i0jhbkhwa) and [support p&p copy doc](https://docs.google.com/document/d/1R7FRZbfx_5YHFeZUD93HgT8RW4OCbbK4blOulB4D6EE/edit#)

## Issue / Card

Fixes #2236
